src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const tables = table.children;
+
+for (const part of tables) {
+  const linescollection = part.children;
+
+  for (const line of linescollection) {
+    let newElement;
+
+    if (part.tagName === 'tbody') {
+      newElement = document.createElement('td');
+    } else {
+      newElement = document.createElement('th');
+    }
+
+    newElement.textContent = line.children[1].textContent;
+
+    line.insertBefore(newElement, line.children[3]);
```

Comment:
Ensure that 'line.children[3]' exists before trying to insert 'newElement' before it. If 'line.children[3]' does not exist, this will throw an error. Consider adding a check or handling for cases where 'line.children[3]' might be undefined.